interfaces/identity.js

```diff
@@ -0,0 +1,22 @@
+type chrome$AccountInfo = string;
+
+type chrome$identity = {
+  getAccounts(callback: (accounts: Array<chrome$AccountInfo>) => void): void,
+  getAuthToken(details: {
+    interactive?: boolean,
+    account?: chrome$AccountInfo,
+    scopes?: Array<string>
+  }, callback?: (token: ?string) => void): void,
+  getProfileUserInfo(callback: (userInfo: { email: ?string, id: ?string }) => void): void,
+  removeCachedAuthToken(details: {
+    token: string
+  }, callback?: () => void): void,
+  launchWebAuthFlow(details: {
+    url: string,
+    interactive?: boolean
+  }, callback: (responseUrl: ?string) => void): void,
+  getRedirectURL(path?: string): string,
```

Comment:
I don't think `path` should be optional

Reply:
`path` is optional in this case, since `getRedirectURL` returns a url and adding `path` saves you from adding manually via `+ "/auth/fb"`.

https://developer.chrome.com/apps/identity#method-getRedirectURL